libraries/package-extractor/src/Utils.ts

```diff
@@ -54,3 +56,48 @@ export function remapPathForExtractorMetadata(folderPath: string, filePath: stri
   }
   return Path.convertToSlashes(relativePath);
 }
+
+/**
+ * Creates the .bin files for the extracted projects and returns the paths to the created .bin files.
+ *
+ * @param terminal - The terminal to write to
+ * @param extractedProjectFolderPaths - The paths to the extracted projects
+ */
+export async function makeBinLinksAsync(
+  terminal: ITerminal,
+  extractedProjectFolderPaths: string[]
+): Promise<string[]> {
+  const binFilePaths: string[] = [];
+  await Async.forEachAsync(
+    extractedProjectFolderPaths,
+    async (extractedProjectFolderPath: string) => {
+      const extractedProjectNodeModulesFolderPath: string = path.join(
+        extractedProjectFolderPath,
+        'node_modules'
+      );
+      const extractedProjectBinFolderPath: string = path.join(extractedProjectNodeModulesFolderPath, '.bin');
+
+      const linkedBinPackageNames: string[] = await pnpmLinkBins(
+        extractedProjectNodeModulesFolderPath,
+        extractedProjectBinFolderPath,
+        {
+          warn: (msg: string) => terminal.writeLine(Colorize.yellow(msg))
+        }
+      );
+
+      if (linkedBinPackageNames.length) {
+        const binFolderItems: string[] = await FileSystem.readFolderItemNamesAsync(
+          extractedProjectBinFolderPath
+        );
+        for (const binFolderItem of binFolderItems) {
+          binFilePaths.push(path.resolve(extractedProjectBinFolderPath, binFolderItem));
```

Comment:
Why resolve?

Reply:
I like resolve more than join in scenarios where the path gets logged out (which really are the only scenarios where we care) since it creates the simplest version of the path using the OS-specific path separators

Reply:
In this case however, the paths should be known to be simplified down enough to not need to resolve vs join